src/sentry/models/organization.py

```diff
@@ -127,6 +127,10 @@ class Organization(Model):
                 "disable_new_visibility_features",
                 "Temporarily opt out of new visibility features and ui",
             ),
+            (
+                "require_email_verification",
+                "Require and enforce email verification for all members.",
+            ),
         ),
         default=1,
```

Comment:
not exactly related to your PR what does having `default=1` do on a BitField? is it automatically set to true then?

Reply:
Bitfield is a bigint, so setting it to 1 just enabled the first bit